src/deploy/functions/triggerRegionHelper.ts

```diff
@@ -7,7 +7,7 @@ import { serviceForEndpoint } from "./services";
  * @param have the list of function specs we have deployed
  */
 export async function ensureTriggerRegions(want: backend.Backend): Promise<void> {
-  const regionLookups: Array<Promise<void>> = [];
+  const regionLookups: Array<Promise<void> | void> = [];
```

Comment:
huh I wonder why this change was necessary 🤔

Reply:
This was added for the typescript compiler. We don't need a lookup for the trigger region in firealerts and the corresponding dispatch function is not async, just returns void

Reply:
Repeating message above:

what if we "clean" this type up by requiring everything to return a promise - it is easy to make non-promise returning fns to return a promise instead.

Reply:
fixed in the latest commit